### PR TITLE
Payoutable implementation

### PR DESCRIPTION
**Pull request by daniel-iobuilders**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,113 @@
...
+}
+```
+
+> You need an ethereum development framework for the above import statements to work! Check out these guides for [Truffle], [Embark] or [Buidler].
+
+## Payment information
+
+Whenever a payout is ordered, payment information has to be provided with the necessary information for the off-chain transfer. [EIP-2021](https://github.com/ethereum/EIPs/pull/2021/files) leaves the structure of this information up to the implementer, but recommends [ISO-20022](https://en.wikipedia.org/wiki/ISO_20022) as a starting point.
+
+The unit tests use a JSON version of this standard, which can be seem below.
```
Comment (Ferparishuertas) [inline README.md:71]: When i mentioned 20022 as an inspiration it was that, an inspiration, not a copy , so I think we need to review it. Having said that, what it is quite important is to not to store any sensible information onchain. I  think i wasnt clear enough but i meant to refer to an payout account that the token operator/owner (.f.e. io.cash), already knows and is refered like a token or similar. account 1, or UUID based

File: contracts/IPayoutable.sol [hunk 2]
```diff
@@ -1,28 +1,45 @@
...
-    event PayoutCancelled(address indexed orderer, string indexed operationId);
-    event PayoutOperatorAuthorized(address indexed walletToBePaidOut, address indexed orderer);
-    event PayoutOperatorRevoked(address indexed walletToBePaidOut, address indexed orderer);
+    event PayoutOrdered(address indexed orderer, string operationId, address indexed walletToDebit, uint256 value, string instructions);
+    event PayoutInProcess(address indexed orderer, string operationId);
+    event PayoutFundsInSuspense(address indexed orderer, string operationId);
+    event PayoutExecuted(address indexed orderer, string operationId);
+    event PayoutRejected(address indexed orderer, string operationId, string reason);
+    event PayoutCancelled(address indexed orderer, string operationId);
+    event AuthorizedPayoutOperator(address indexed operator, address indexed account);
```
Comment (Ferparishuertas) [inline contracts/IPayoutable.sol:43]: Why do we do this naming change when the original was good , in a paste sense as a normal event?

File: contracts/IPayoutable.sol [hunk 3]
```diff
@@ -1,28 +1,45 @@
...
+    event RevokedPayoutOperator(address indexed operator, address indexed account);
```
Comment (Ferparishuertas) [inline contracts/IPayoutable.sol:44]: Why do we do this naming change when the original was good , in a paste sense as a normal event?

File: contracts/IPayoutable.sol (hunk 2 above)
Comment (daniel-iobuilders) [reply contracts/IPayoutable.sol:43]: In the Holdable and Clearable the naming of the events is done with `AuthorizedHoldOperator` and `AuthorizedClearableTransferOperator`. This naming is copied from ERC-777.
I don't mind which format we use, but it should be consistent across our EIPs. So if you prefer `PayoutOperatorAuthorized` I will change it in the other EIPs as well.

**Pull request merged**